recipes/ufmt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ufmt" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ufmt-{{ version }}.tar.gz
+  sha256: 7e4518e13507984998f5c7c9a0d3f59fe014ddc213bdb5cae8d11a6a8935db8c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit >=3,<4
+    - black >=20.8b0
+    - moreorless >=0.4.0
+    - trailrunner >=1.2.1
+    - usort >=1.0
+    - tomlkit >=0.7.2
```

Comment:
These are probably not needed. If they are the package imports itself on build time and that is, most of the time, a bug upstream.

```suggestion
```

Reply:
I assume this is caused by flit (I iteratively added packages until build was succeeding ...). Agreed that this seems like an upstream bug to me. Does this need to get fixed before we can submit a package?

Reply:
Indeed flit attempts to load the modules in order to extract the version from the __init__.py field. There is an AST parsing based extractor, however the __version__ field is imported from a separate file. I am working around this by inlining the version string into the toplevel directory.